smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaSource.java

```diff
@@ -264,6 +275,36 @@ public synchronized void reportFailure(Throwable failure) {
         failures.add(failure);
     }
 
+    public void incomingTrace(IncomingKafkaRecord<K, V> kafkaRecord) {
+        if (configuration.getTracingEnabled()) {
```

Comment:
The signature looks weird `getTracingEnabled()` it could be `.isTracingEnabled()`?

Reply:
I think that's just how the methods are generated, as it's not manually coded